test.sh

```diff
@@ -378,3 +378,24 @@ orbit/warpdrive.sh -u user -w
 orbit/warpdrive.sh -r \
   | tee test/roster_withdrawn_empty \
   | diff /dev/stdin <(echo "Users:")
+
+# Verify that we cannot delete a user a second time
+# We expect that this will fail so putting ! in front makes succeeedin
```

Comment:
same missin a g typo 3 more times here

Reply:
Maybe drop the repeated comment and make a single announcement at the start of this block or a the top about how testing for an expected fail works in this script to keep things DRY

Reply:
yeah, will refactor.